### PR TITLE
GDB-8522 Enable Sonar analysis in Jenkins

### PR DESCRIPTION
**Pull request by yordanalexandrov**
## What
Enable code static analysis

## Why
So that code is of good quality

## How
- enable jenkins stage for sonarcloud

### CODE REVIEW

**Comment by yordanalexandrov** [comment]
Jenkins retry build

**Comment by yordanalexandrov** [comment]
Jenkins retry build

**Comment by yordanalexandrov** [comment]
Jenkins retry build

**Comment by yordanalexandrov** [comment]
Jenkins retry build

File: Jenkinsfile [hunk 1]
```diff
@@ -7,9 +7,9 @@ pipeline {
 // Doesn't work on Jenkins due to some errors in GDB responses. Locally it works though.
 // It works fine using the default node version in jenkins node which is 12.19.0, but SonarCloud
 // doesn't like it.
```
Comment (svilenvelikov) [inline Jenkinsfile:9]: Remove this comment also the one for the `Sonar` stage

**Review by svilenvelikov — APPROVED** [review]

**Pull request merged**